scripts/explore.py

```diff
@@ -209,6 +248,89 @@ def gatherTimes():
     os.system(addHeader)
     os.chdir(explorationDir)
 
+#global exploration length in mins
+explorationLength = 0
+def saveExplorationMetaInformation():
+    global explorationLength
+    os.chdir(explorationDir)
+    kernelNumber = "cd "+explorationDir+"/"+expressionCl+";  ls */*.cl | wc -l"
+    validExecutions = "find "+explorationDir+"/"+expressionCl+" -name \"" + timeCsv + "\" | xargs cat | wc -l"
```

Comment:
if you look in the harness repo you find that besides `time_X.csv` also files like `blacklist.csv` and `compilationerror.csv` might get created. It would be good to count these entries as well and in case the file does not exist list 0